include/modem/lte_lc.h

```diff
@@ -464,6 +464,16 @@ enum lte_lc_modem_evt {
 	LTE_LC_MODEM_EVT_OVERHEATED,
 };
 
+/** @brief type of factory reset to perform */
+enum lte_lc_factoryreset_type {
+	/** Reset all modem data to factory settings */
```

Comment:
```suggestion
	/** Reset all modem data to factory settings. */
```

Reply:
Nit: Really not necessary, but nice to align with the other documentation in this header file.

Reply:
This also goes for the other comments as well.